web_widget_prefixed_url/README.rst

```diff
@@ -0,0 +1,37 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :alt: License: AGPL-3
+
+========================================================
+Module for adding 'prefixed_url:' widget to your fields.
```

Comment:
Change 'prefixed_url:' to 'prefixed_url' or can you change it to the module name?
I prefer using the readme generator but my understanding is that it's not required.

Reply:
Good Catch :+1: 

I changed it to `prefixed_url`